ports/mimxrt/boards/MIMXRT1050_EVK/mpconfigboard.h

```diff
@@ -1,14 +1,10 @@
-#define MICROPY_HW_BOARD_NAME "i.MX RT1050 EVK"
+#define MICROPY_HW_BOARD_NAME "i.MX RT1050 EVKB"
```

Comment:
It looks like the board dir got renamed from MIMXRT1050_EVKB to MIMXRT1050_EVK, yet this name here is the opposite, the B got added, why?

Reply:
fixed.